alws/build_planner.py

```diff
@@ -230,16 +283,10 @@ async def add_task(self, task: build_schema.BuildTaskRef):
                 module.set_arch_list(
                     self._request_platforms[platform.name]
                 )
-                self.remove_force_to_build_artifacts_from_template(
-                    module, raw_refs)
-                module_index = IndexWrapper()
                 module_index.add_module(module)
-                if len(module_templates) > 1:
-                    devel_module = ModuleWrapper.from_template(
-                        module_templates[1],
-                        module.name + '-devel',
-                        module.stream
-                    )
+                if module_index.has_devel_module():
+                    devel_module = module_index.get_module(
+                        f'{task.module_name}', task.module_stream)
```

Comment:
does we need here `-devel` suffix? or it's works fine for devel modules?

Reply:
Yep, you're right, missed it